modules/internal/String.chpl

```diff
@@ -94,12 +112,24 @@ module String {
   pragma "ignore noinit"
   pragma "no default functions" // avoid the default (read|write)This routines
   record string {
+    pragma "no doc"
     var len: int = 0; // length of string in bytes
+    pragma "no doc"
     var _size: int = 0; // size of the buffer we own
+    pragma "no doc"
     var buff: bufferType = nil;
+    pragma "no doc"
     var owned: bool = true;
-    var locale_id /* : chpl_nodeID_t */ = chpl_nodeID;
-
+    pragma "no doc"
+    var locale_id = chpl_nodeID; // : chpl_nodeID_t
```

Comment:
Is there a reason we can't specify the type here? If it's a bug, can you note that in a todo